OpenSim/Common/Test/testDataTable.cpp

```diff
@@ -251,6 +268,29 @@ int main() {
             ASSERT(row2[8] == 1);
         }
 
+        {
+        std::cout << "Test feeding rows of one table to another [Vec3]."
+                  << std::endl;
+        DataTableVec3 tableVec3Copy{};
+        tableVec3Copy.setColumnLabels(tableVec3.getColumnLabels());
+        for(unsigned row = 0; row < tableVec3.getNumRows(); ++row)
+            tableVec3Copy.appendRow(tableVec3.getIndependentColumn()[row],
+                                    tableVec3.getRowAtIndex(row));
+
+        ASSERT(tableVec3Copy.getNumColumns() == tableVec3.getNumColumns());
+        ASSERT(tableVec3Copy.getNumRows()    == tableVec3.getNumRows());
+        for(unsigned r = 0; r < tableVec3.getNumRows(); ++r) {
+            for(unsigned c = 0; c < tableVec3.getNumColumns(); ++c) {
+                ASSERT(tableVec3Copy.getRowAtIndex(r).getElt(0, c)[0] ==
+                           tableVec3.getRowAtIndex(r).getElt(0, c)[0]);
+                ASSERT(tableVec3Copy.getRowAtIndex(r).getElt(0, c)[1] ==
+                           tableVec3.getRowAtIndex(r).getElt(0, c)[1]);
+                ASSERT(tableVec3Copy.getRowAtIndex(r).getElt(0, c)[2] ==
+                           tableVec3.getRowAtIndex(r).getElt(0, c)[2]);
```

Comment:
I thought ASSERT_EQUAL without a tolerance was a bitwise match, but it looks like in that case it defaults to some tolerance, so you can leave it as you have it.

It is troubling that the `[ ]` operator does not work in debug. That should be an issue, not for this PR. What assertion is failing?  Do you know why? If user's have to access elements of a row using `.getElt()` all the time that is going to be ☹️ for them.